modules/FhirAPI/src/FhirAPI/FhirRestApiBuilder/Parts/Search/SearchStrategies/ObservationSearch.php

```diff
@@ -23,9 +23,15 @@ public function search()
         $this->paramHandler('status','activity');
         $this->paramHandler('encounter','eid');
         $this->paramHandler('category','category');
+        
+        $this->addSortParams('issued','date');
```

Comment:
we have more places in the other entities without a new function -  addSortParams you added.
Make sure this fix does not destroy other sort parameres